examples/tilejson.js

```diff
@@ -8,7 +8,8 @@ var map = new ol.Map({
   layers: [
     new ol.layer.Tile({
       source: new ol.source.TileJSON({
-        url: 'http://api.tiles.mapbox.com/v3/mapbox.geography-class.json',
+        jsonp: true,
+        url: 'http://api.tiles.mapbox.com/v3/mapbox.geography-class.jsonp',
```

Comment:
Oops, thanks for spotting this, it's an uniintentional change. I had just changed it for testing.